src/test/java/com/rackspace/jenkins_nodepool/UtilsTest.java

```diff
@@ -0,0 +1,4 @@
+package com.rackspace.jenkins_nodepool;
+
+public class UtilsTest {
```

Comment:
Guessing this isn't needed either as the utils class is empty.

Reply:
Removed.